### PR TITLE
Fix an error for `RSpecRails/HttpStatus`

### PR DESCRIPTION
**Pull request by ydah**
Fix: rubocop/rubocop-rspec_rails#9

______________________________________________________________________

Before submitting the PR make sure the following are checked:

- [x] Feature branch is up-to-date with `master` (if not - rebase it).
- [x] Squashed related commits together.
- [x] Added tests.
- [-] Updated documentation.
- [x] Added an entry to the `CHANGELOG.md` if the new code introduces user-observable changes.
- [x] The build (`bundle exec rake`) passes (be sure to run this locally, since it may produce updated documentation that you will need to commit).

### CODE REVIEW

**Comment by ntkme** [comment]
You might want to unconditionally load this if rack is going to be a dependency that is always available: https://github.com/rubocop/rubocop-rspec_rails/blob/b141576cf6fb1f75bb3e446b96d4c4f8266cc161/lib/rubocop/cop/rspec_rails_cops.rb#L6-L10

File: Gemfile [hunk 1]
```diff
@@ -15,7 +14,7 @@ gem 'simplecov', '>= 0.19'
 gem 'yard'
 
 # TODO: Move to gemspec when RuboCop RSpec v3 is released.
-gem 'rubocop-rspec', '~> 2.27'
+gem 'rubocop-rspec', '~> 2.27.1'
```
Comment (ydah) [inline Gemfile:17]: The following error occurs if you do not lock with version 2.27.1

```
❯ bundle exec rake
rake aborted!
NameError: uninitialized constant RuboCop::Cop::RSpecRails (NameError)

        AvoidSetupHook = ::RuboCop::Cop::RSpecRails::AvoidSetupHook
                                       ^^^^^^^^^^^^
/ydah/rubocop-rspec_rails/lib/rubocop-rspec_rails.rb:7:in `<top (required)>'
tasks/cops_documentation.rake:4:in `<top (required)>'
/ydah/rubocop-rspec_rails/Rakefile:19:in `load'
/ydah/rubocop-rspec_rails/Rakefile:19:in `block in <top (required)>'
/ydah/rubocop-rspec_rails/Rakefile:19:in `each'
/ydah/rubocop-rspec_rails/Rakefile:19:in `<top (required)>'
/.rbenv/versions/3.3.0/bin/bundle:25:in `load'
/.rbenv/versions/3.3.0/bin/bundle:25:in `<main>'
(See full trace by running task with --trace)
```

**Comment by pboling** [comment]
This may fix #10 as well, or perhaps it is a duplicate issue of #9 ?

File: rubocop-rspec_rails.gemspec [hunk 2]
```diff
@@ -34,5 +34,6 @@ Gem::Specification.new do |spec|
     'rubygems_mfa_required' => 'true'
   }
 
+  spec.add_runtime_dependency 'rack'
```
Comment (bquorning) [inline rubocop-rspec_rails.gemspec:37]: I think we should only add this dependency once we have released a few versions of rubocop-rspec that don’t depend on rubocop-rspec_rails.

**Pull request closed without merging**